engine/servermaster/jobmanager.go

```diff
@@ -107,6 +113,83 @@ func (jm *JobManagerImplV2) PauseJob(ctx context.Context, req *pb.PauseJobReques
 	}}
 }
 
+// DebugJob implements proto/Master.DebugJob
+// Used for request/response with jobmaster
+func (jm *JobManagerImplV2) DebugJob(ctx context.Context, req *pb.DebugJobRequest) (resp *pb.DebugJobResponse) {
```

Comment:
Will this be refactored or removed after OpenAPI is finished.

Reply:
Yes